### ISSUE TITLE
Return VICRegLoss components

### ISSUE THREAD

**Issue by md-121** [lightly-ai/lightly#1161]
Hi everyone,

the `VICRegLoss` implementation returns the final weighted sum of the three loss components (Invariance loss, Variance loss and Covariance loss).
I would like to see/log the individual components of the VICRegLoss as I think this could give valuable insights into the training of models.

Thank you :)

**Comment by guarin** [comment]
Hi :)
You can calculate the individual loss components like this and log them before calculating the total loss:

```
from lightly.loss.vicreg_loss import invariance_loss, variance_loss, covariance_loss

inv_loss = invariance_loss(x=z_a, y=z_b)
var_loss = 0.5 * (variance_loss(x=z_a) + variance_loss(x=z_b))
cov_loss = covariance_loss(x=z_a) + covariance_loss(x=z_b)
print(inv_loss, var_loss, cov_loss)
total_loss = 25.0 * inv_loss + 25.0 * var_loss + 1.0 * cov_loss
```

Hope this helps :)

**Comment by md-121** [comment]
Oh, yes sure!
Not sure why I missed to see this easy solution 😅 

Thanks for the quick reply @guarin 😄
